skimage/transform/_warps.py

```diff
@@ -190,16 +216,14 @@ def rescale(image, scale, order=1, mode=None, cval=0, clip=True,
     (256, 256)
 
     """
-
-    try:
-        row_scale, col_scale = scale
-    except TypeError:
-        row_scale = col_scale = scale
-
-    orig_rows, orig_cols = image.shape[0], image.shape[1]
-    rows = np.round(row_scale * orig_rows)
-    cols = np.round(col_scale * orig_cols)
-    output_shape = (rows, cols)
+    multichannel = _multichannel_default(multichannel, image.ndim)
+    scale = np.atleast_1d(scale)
+    if len(scale) > 1 and len(scale) != image.ndim:
+        raise ValueError("must supply a single scale or one value per axis.")
```

Comment:
Wouldn't there be a third case — for multichannel images, len(scale) == image.ndim[:-1], right?